devops/poseidon-deployer/tasks/deployPoseidon.ts

```diff
@@ -0,0 +1,96 @@
+import { task, subtask } from "hardhat/config";
+import { Contract } from "ethers";
+
+import { verifyPoseidonContracts } from "./utils";
+
+
+task("deploy-poseidon", "Deploy poseidon contracts by sending multiple txs or calling multicall")
+.addFlag("multicall", "Use this flag to invoke multicall")
```

Comment:
Remove this flag add a flag for each PoseidonT[x] supported

Reply:
Created `deploy-poseidon:callPoseidonFactory` command to deploy each PoseidonT[x] by multicall:

https://github.com/privacy-scaling-explorations/maci/blob/3f753a1792931a5910c3614db2426bef2bb134c1/devops/poseidon-deployer/tasks/deployPoseidon.ts#L29